test/server/cards/03_TWI/units/PerilousPosition.spec.ts

```diff
@@ -0,0 +1,36 @@
+describe('Confederate Courier', function () {
+    integration(function (contextRef) {
+        describe('ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['perilous-position'],
+                        groundArena: ['outspoken-representative']
+                    },
+                    player2: {
+                        groundArena: [{ card: 'duchesss-champion', exhausted: true }]
+                    },
+                });
+            });
+
+            it('targeted unit should be exhausted when played', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.perilousPosition);
+                expect(context.player1).toBeAbleToSelectExactly([context.outspokenRepresentative, context.duchesssChampion]);
+                context.player1.clickCard(context.outspokenRepresentative);
+                expect(context.outspokenRepresentative.exhausted).toBeTrue();
+            });
+
+            it('targeting exhausted unit should still be exhausted when played', function () {
```

Comment:
Good idea but I think we're okay without this test, we have lots of tests confirming that the exhaust operation is idempotent so for the of keeping test time down I think we can remove this one (and remove the Duchess's Champion unit from the test setup)